src/private/index.ts

```diff
@@ -26,4 +26,4 @@ export {
 } from './privateAPIs';
 export { conversations } from './conversations';
 export { meetingRoom } from './meetingRoom';
-export { meetingAudio } from './meetingAudio';
+export { meeting } from './meeting';
```

Comment:
It seems like these APIs will be exposed publicly so you might want to move them under `src/public` folder instead of `src/private` where they are today.